src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetSpecialVideosTests.java

```diff
@@ -38,7 +38,7 @@ public void SpecialVideos_001_Provider() {
 
     FilePagePageObject filePage = new FilePagePageObject(driver).open(video.getWikiFileName());
 
-    filePage.getGlobalNavigation().openAccountNavigation().clickLogOut();
+//    filePage.getGlobalNavigation().openAccountNavigation().clickLogOut();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)